src/commands/cronjob/create.tsx

```diff
@@ -87,23 +79,18 @@ export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
       throw new Error("no project found for app installation");
     }
 
-    const destination: CronjobCreationData["destination"] =
-      buildCronjobDestination(url, command, interpreter);
-
-    const cronjobCreationData: CronjobCreationData = {
-      appId: appInstallationId,
-      active: !disable,
-      description,
-      interval,
-      email,
-      destination,
-      timeout: timeout.seconds,
-    };
-
     const { id: cronjobId } = await p.runStep("creating cron job", async () => {
       const r = await this.apiClient.cronjob.createCronjob({
         projectId,
-        data: cronjobCreationData,
+        data: {
+          appId: appInstallationId,
+          active: !disable,
+          description,
+          interval,
+          email,
+          destination: buildCronjobDestination(url, command, interpreter),
+          timeout: timeout.seconds,
+        },
```

Comment:
For some reason I was under the impression that if a key has an undefined value, currently set values for conjobs (or whichever entity is being updated) will be overwritten, I played around for a bit and it seems to not be the case wich makes this approach the most concise.